syslogish/server.go

```diff
@@ -16,7 +16,7 @@ const (
 	bindHost          = "0.0.0.0"
 	bindPort          = 1514
 	queueSize         = 500
-	controllerPattern = `^(INFO|WARN|DEBUG|ERROR)\s+(\[(\S+)\])+:(.*)`
+	controllerPattern = `(INFO|WARN|DEBUG|ERROR)\s+(\[(\S+)\])+:(.*)`
```

Comment:
I don't understand how this change would remove duplicate lines. The `^` character anchors the regex match to the beginning of a line, which is *more* specific than without it.

I'm sure you tested this and it's a fix, but I must be missing something by just reading it.